db/migrate/20160620141513_add_webhook_to_organizations.rb

```diff
@@ -0,0 +1,6 @@
+class AddWebhookToOrganizations < ActiveRecord::Migration
+  def change
+    add_column :organizations, :webhook_id, :string, unique: true
```

Comment:
Thanks for the catch! Actually the `unique: true` has no effect since it's an invalid option for `add_column` 😓 

Reference: http://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-add_column